crates/proof_of_stake/src/tests/test_pos.rs

```diff
@@ -385,7 +385,7 @@ fn test_bonds_aux(params: OwnedPosParams, validators: Vec<GenesisValidator>) {
         assert_eq!(
             details.bonds.len(),
             2,
-            "Contains genesis and newly added self-bond"
+            "contains_key genesis and newly added self-bond"
```

Comment:
Are these modifications indeed correct? They look like the result of a search&replace but I'm not sure they were intended